content/en/integrations/observability_pipelines/integrate_vector_with_datadog.md

```diff
@@ -0,0 +1,608 @@
+
+{{% /tab %}}
+{{% tab "TOML" %}}
+
+```toml
+sinks:
+  metrics_to_datadog:
+    type: datadog_metrics
+    inputs:
+      - tag_metrics
+    default_api_key: '${DATADOG_API_KEY_ENV_VAR}'
+   compression: gzip
+```
+
+{{% /tab %}}
+{{% tab "JSON" %}}
+
+```json
+{
+  "sinks": {
+    "metrics_to_datadog": {
+      "type": "datadog_metrics",
+      "inputs": [
+        "tag_metrics"
+      ],
+      "default_api_key": "${DATADOG_API_KEY_ENV_VAR}",
+    compression: gzip
+   }
+  }
+}
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+#### Traces
+
+To send traces to Datadog, Vector must be configured with a [datadog_traces sink][16]. See the following example:
+
+{{< tabs >}}
+{{% tab "YAML" %}}
+
+```yaml
+sinks:
+  traces_to_datadog:
+    type: datadog_traces
+    inputs:
+       - component_id
+    default_api_key: "${DATADOG_API_KEY_ENV_VAR}"
+```
+
+{{% /tab %}}
+{{% tab "TOML" %}}
+
+```toml
+sinks:
+  traces_to_datadog:
+    type: datadog_traces
+    inputs:
+      - component_id
+    default_api_key: '${DATADOG_API_KEY_ENV_VAR}'
+```
+
+{{% /tab %}}
+{{% tab "JSON" %}}
+
+```json
+{
+  "sinks": {
+    "traces_to_datadog": {
+      "type": "datadog_traces",
+      "inputs": [
+        "component_id"
+      ],
+      "default_api_key": "${DATADOG_API_KEY_ENV_VAR}"
+   }
+  }
+}
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+## Advanced Vector configurations
+
+### Disk buffers
+
+Datadog recommends enabling disk buffers to prevent data loss. Vector uses [disk buffers][17] to ensure no data is lost when there is a spike in data being sent or the downstream service is sending back pressure.  See the configuration below for setting buffers at the sink level.
+
+{{< tabs >}}
+{{% tab "YAML" %}}
+
+```yaml
+sinks:
+  metrics_to_datadog:
+    type: datadog_metrics
+    inputs:
+       - tag_metrics
+    default_api_key: "${DATADOG_API_KEY_ENV_VAR}"
+    compression: gzip
+    buffer:
+         type: disk
+         max_size: 309237645312
+```
+
+{{% /tab %}}
+{{% tab "TOML" %}}
+
+```toml
+[sinks.metrics_to_datadog]
+type = "datadog_metrics"
+inputs = [ "some_input_id" ]
+default_api_key = "${DATADOG_API_KEY_ENV_VAR}"
+compression = "gzip"
+
+  [sinks.metrics_to_datadog.buffer]
+  type = "disk"
+  max_size = 309_237_645_312
+
+```
+
+{{% /tab %}}
+{{% tab "JSON" %}}
+
+```json
+{
+  "sinks": {
+    "metrics_to_datadog": {
+      "type": "datadog_metrics",
+      "inputs": [
+        "tag_metrics"
+      ],
+      "default_api_key": "${DATADOG_API_KEY_ENV_VAR}",
+    compression: gzip,
+    “buffer”: {
+         “type”: “disk”
+         “max_size”: 309237645312
+     }
+   }
+  }
+}
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+#### Disk space
+
+You should provision at least 36 GiB per vCPU * of disk space. If you follow the recommendation of 8 vCPUs, you would provision 288 GiB of disk space (10 MiB * 60 seconds * 60 minutes * 8 vCPUs), allocating ​​48 GiB for metrics and 240 GiB for logs. You can add a volume to the Vector instances to hold the buffer in your Helm chart:
```

Comment:
```suggestion
You should provision at least 36 GB per vCPU * of disk space. If you follow the recommendation of 8 vCPUs, you would provision 288 GB of disk space (10 MB * 60 seconds * 60 minutes * 8 vCPUs), allocating ​​48 GB for metrics and 240 GB for logs. You can add a volume to the Vector instances to hold the buffer in your Helm chart:
```

Reply:
Thanks @urseberry! It seems like there is a slight difference between GiB and GB measurements: https://kb.wisc.edu/researchdata/page.php?id=101840

I'll defer to @barieom on whether this should be changed to GB.